test/server/lib/GranularAccess.ts

```diff
@@ -3547,6 +3548,16 @@ describe('GranularAccess', function() {
       {id: [1], MorePics: [[GristObjCode.List, i6]]}
     ), /403.*Cannot access attachment/);
 
+    const readTokenResult = (await cliEditor.send("getAccessToken", 0, {readOnly: true})).data;
+    await assert.isRejected(postAttachment(editor, docId, readTokenResult.token, "dataError", "error.txt"));
+
+    const tokenResult = (await cliEditor.send("getAccessToken", 0, {readOnly: false})).data;
+    const i7 = await postAttachment(editor, docId, tokenResult.token, "data7", "7.txt");
+    await assert.isFulfilled(getAttachment(owner, docId, i7));
+    //await assert.isFulfilled(getAttachment(editor, docId, i7));
```

Comment:
This should be discussed, it fails because it is impossible to set maybeNew from the web api. To me, in should be made possible. I can open another PR for that.